### PR TITLE
Switch CI to GitHub Actions

### PR DESCRIPTION
**Pull request by makaimann**

### CODE REVIEW

**Comment by makaimann** [comment]
Marked `do not merge`, because the Travis CI hooks should be removed and the `.travis.yml` file deleted before merging.

**Review by lonsing — APPROVED** [review]
Looks good to me, thanks!

**Comment by makaimann** [comment]
Thanks for the review!

**Pull request merged**
